Common/Helpers/AzureTableStorageHelper.cs

```diff
@@ -53,15 +50,15 @@ public static CloudTable GetTable(string storageConnectionString, string tableNa
                 operation = TableOperation.Insert(incomingEntity);
             }
 
-            return await PerformTableOperation<TResult, TInput>(table, operation, incomingEntity, tableEntityToModelConverter);
+            return await PerformTableOperation(table, operation, incomingEntity, tableEntityToModelConverter);
         }
 
         public static async Task<TableStorageResponse<TResult>> DoDeleteAsync<TResult, TInput>(TInput incomingEntity,
             Func<TInput, TResult> tableEntityToModelConverter, string storageAccountConnectionString, string tableName) where TInput : TableEntity
         {
             var azureTable = await AzureTableStorageHelper.GetTableAsync(storageAccountConnectionString, tableName);
             TableOperation operation = TableOperation.Delete(incomingEntity);
-            return await PerformTableOperation<TResult, TInput>(azureTable, operation, incomingEntity, tableEntityToModelConverter);
+            return await PerformTableOperation(azureTable, operation, incomingEntity, tableEntityToModelConverter);
         }
 
         private static async Task<TableStorageResponse<TResult>> PerformTableOperation<TResult, TInput>(CloudTable table, 
```

Comment:
it's still the same function. Since DoDeleteAsync has the generics defined in its signature: DoDeleteAsync<TResult, TInput> the argument specification is redundant in PerformTableOperation<TResult, TInput>. I'm just removing redundant arguments, nothing is changed.